examples/nidaqmx - time tased synchronization.py

```diff
@@ -0,0 +1,34 @@
+import nidaqmx
+import time
+from nidaqmx.constants import AcquisitionType
+from datetime import timedelta, datetime, timezone
+
+# Function to perform the measurement
+def perform_measurement():
+    with nidaqmx.Task() as task:
+        task.ai_channels.add_ai_voltage_chan("Dev1/ai0")
+        
+        # Configure finite sampling
+        num_samples = 1000
+        task.timing.cfg_samp_clk_timing(1000, sample_mode=AcquisitionType.FINITE, samps_per_chan=num_samples)
+        
+        # Calculate the start time (1 minute from now)
+        start_time = datetime.now(timezone.utc) + timedelta(minutes=1)

```

Comment:
1 minute is a lot...

```suggestion
        # Calculate the start time (10 seconds from now)
        start_time = datetime.now(timezone.utc) + timedelta(seconds=10)
```